src/overlays/actors/ovl_En_Pu_box/z_en_pu_box.c

```diff
@@ -59,7 +59,7 @@ void EnPubox_Init(Actor* thisx, GlobalContext* globalCtx) {
     this->dyna.unk_15C = 0;
     thisx->unk_1F = 1;
     thisx->gravity = -2.0f;
-    DynaPolyInfo_Alloc(&D_060006D0, &local_c);
+    DynaPolyInfo_Alloc(&puBoxDynaPolyInfo, &local_c);
```

Comment:
pretty sure the terminology we are going with for these is "collision header", but if you want to wait until bgcheck is merged to update those names thats fine too